src/NuGetGallery.Services/Configuration/CacheConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGetGallery.Services
+{
+    public class CacheConfiguration : ICacheConfiguration
+    {
+        public const int DefaultGetPackageDependentsCacheTimeInSeconds = 3600;
```

Comment:
Super nit: consider removing the "Get" verb here since it doesn't add too much value.

```suggestion
        public const int DefaultPackageDependentsCacheTimeInSeconds = 3600;
```

I don't feel strongly about this, feel free to keep it as-is if you prefer it :)